### PR TITLE
doc : fixes issue #2327

### PR DESCRIPTION
**Pull request by tashiya-menuri**
### What this PR does / why we need it:
Fix for https://github.com/apache/apisix/issues/2327

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Reference Documentation
 * [Router radixtree](router-radixtree.md)
 * [Stand Alone Model](stand-alone.md): Supports to load route rules from local yaml file, it is more friendly such as under the kubernetes(k8s).
 * [Stream Proxy](stream-proxy.md)
+* [gRPC Proxy](grpc-proxy.md)
```
Comment (membphis) [inline doc/README.md:34]: Do we need to update the Chinese version?

Comment (tashiya-menuri) [reply doc/README.md:34]: Addressed this in latest commit.

**Comment by tashiya-menuri** [comment]
@moonming @spacewander @membphis Can you please check now?

File: doc/grpc-proxy.md [hunk 2]
```diff
@@ -17,7 +17,7 @@
 #
 -->
 
-[中文](zh-cn/grpc-proxy.md)
+[Chinese](zh-cn/grpc-proxy.md)
```
Comment (spacewander) [inline doc/grpc-proxy.md:20]: It would be better if you leave this unchanged.

**Comment by membphis** [comment]
@tashiya-menuri one for one thing.

fix `doc: add grpc-proxy.md link in doc/README.md #2327` is enough.

File: doc/grpc-proxy.md (hunk 2 above)
Comment (tashiya-menuri) [reply doc/grpc-proxy.md]: +1

**Comment by tashiya-menuri** [comment]
@membphis @spacewander Addressed the requested changes in latest commit. Can you please have a look now?

**Review by moonming — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**